gen/__init__.py

```diff
@@ -492,6 +492,7 @@ def get_dcosconfig_source_target_and_templates(
         'package_ids',
         'template_filenames',
         'enable_windows_agents',
+        'zk_address',  # TODO needs to be relocated in config yaml file as a placeholder
```

Comment:
Hi @jongiddy 
Need your advice on dilemma disccovered. In order to populate new variable `zk_address`, it needs to be added either in 
1) here at get_dcosconfig_source_target_and_templates() directly 
or 
2) dcos-config.yaml.
I and @mediapills believe that 2nd would be more logical, however, where is to allocate {{ zk_address }} in dcos-config.yaml exactly, as the parameter will be NOT used by any of the configs, in-fact. It requires for mesos-agent.exe startup command ONLY which is picked from nssm config, and has no relation with dcos-config.yaml
See example, please: 
```
mesos-agent.exe --master=zk://{{ zk_address }}/mesos

#static
mesos-agent.exe --master=zk://172.16.0.1:2181,172.16.0.2:2181,172.16.0.3:2181/mesos
#dynamic
mesos-agent.exe --master=zk://int-sm-ee-d2iq-66812-1-7d0e3291b8692e6e.elb.us-east-1.amazonaws.com:2181/mesos
```

Reply:
Following our discussion, preferably, the generation of the variable is in `calc.py` and, even if the variable is not used apart from in the Mesos package, we can force the variable to be calculated by creating a dummy file in `dcos-config-windows.yaml`, as shown in my previous attempt at this problem: https://github.com/dcos/dcos/pull/7101/files#diff-3a176d7226c4a814bad89c707c845253

Reply:
@jongiddy recommended to workaround the need by using dummy.txt at Windows template.